src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignmentRegion.java

```diff
@@ -96,28 +112,37 @@ int endOfAlignmentInContig() {
         return assembledContigLength - SVVariantCallerUtils.getNumClippedBases(false, forwardStrandCigar);
     }
 
+    // TODO: 11/27/16 test
+    /**
+     * @return  A packed String representation of this AlignmentRegion, noticeably the fields are not separated by tab.
+     *          Note that the format is NOT the same as that used in {@link #toString()}.
```

Comment:
Nope. It was there and I didn't remove it, and called it while outputting debugging information. Will remove.